### DISCUSSION TITLE
Psychoclown NPC does not have talents

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14004] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Psychoclown NPC that rarely spawns in beacons does not have talents

You can control the clown and use `givetalent "Enroll into Clown College"` and the clown will get the talent

Possible reasons:
- Issue with event/statuseffect 
- Job is "killer" (can't unlock talents?)

_No clown power_
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/b93af5c6-33c0-4f89-aa7e-09e13d4f3371)

Also please give the poor clown some more talents (he's weak), and the clownmaskunique instead of regular clown mask (helps with clown quest) 

talentidentifiers="chonkyhonks,psychoclown,waterprankster,enrollintoclowncollege,**stonewall,skedaddle,exampleofhealth,swole**"

### Reproduction steps

1. Load up a level with beacon
2. `triggerevent beaconclown`
3. control the clown
4. behold the lack of talents
5. `givetalent "Enroll into Clown College"`
6. Clown Power activates

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
To me this seems more like a feature request (to give the clown some talents), not a bug report.

**Reply by ubertpendragon** [reply]
The bug is that the clown doesn't have talents given by the event

**Reply by ubertpendragon** [reply]
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/089a00cb-7ef5-429d-bf3b-fe9e87657844)

**Reply by ubertpendragon** [reply]
Note that I'm suggesting talents only because the clown itself will die very fast even after you fix the bug, it's not much of a threat

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/c8e47f707165c665a1ca1b91d5806c5fb892217f by removing the statuseffectaction from the event (these talents don't really do much anyway, not worth the trouble to make them work imo).
